tools/topology/topology2/cavs/cavs-sdw.conf

```diff
@@ -134,6 +152,37 @@ Object.Pipeline {
 		}
 	}
 
+	host-copier-gain-mixin-playback."3" {
+		index 3
+
+	Object.Widget.copier.1 {
+		stream_name "volume playback 1"
+	}
+	Object.Widget.gain.1 {
+		Object.Control.mixer.1 {
+			name '3 Playback Volume 0'
```

Comment:
we really need to have meaningful PGA control names.

Reply:
ack - the tplg compiler needs to be clever enough to name accordingly i.e. base class names would be templates where instance names (from toplevel) would be used. This would become
```Playback Volume $``` where $ could be the instance name like ```Headphones```